packages/mapstore-events-tracker/js/components/FeaturesTable.js

```diff
@@ -91,6 +92,11 @@ function FeaturesTable({
                                     })}
                                 </svg>
                             </td>
+                            <td>
+                                <a href={`#/?feature-viz=detail&feature=${row.pct.value}`}>

```

Comment:
@tonym3nsa I'm not sure we need an additional button because when you select multiple categories this will be hidden at the end.
The request of the issue is to only switch to the panel so please remove this additional link for the moment.
We should use the same `selectedHref` so we should modify this selector by adding the `feature-viz=detail` query param.
https://github.com/geosolutions-it/events-tracker/blob/main/packages/mapstore-events-tracker/js/selectors/viewer.js#L198

```js
selectedHref: hashLocationToHref({
  location,
  query: {
      feature: value,
      'feature-viz': 'detail'
  }
})
```